libsignal-protocol/src/keys/public.rs

```diff
@@ -103,7 +105,16 @@ mod tests {
 
     #[test]
     fn decode_from_binary() {
```

Comment:
Instead of breaking a test build with `compile_error!()` when someone runs `cargo test --no-default-features`, this test should probably have a `#[cfg(any(feature = "crypto-native", feature = "crypto-openssl"))]`.

Reply:
done